initdb.py

```diff
@@ -0,0 +1,5 @@
+from mailbbgun.models import db
+
+db.create_all()
+
+print("mailbbgun database schema initialized.")
```

Comment:
Are you a fan of out of band db migrations or should the application be responsible for this and why?